src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,31 @@
+.nav {
+  display: block;
+
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+  &__item {
+    margin-left: 20px;
+  }
+  &__item:first-child {
+    margin-left: 0;
+  }
```

Comment:
please follow the BEM structure
![Screenshot 2023-06-03 at 19 03 13](https://github.com/mate-academy/layout_catalog/assets/64746943/51cd5baa-542a-4ba7-8048-875faac73f94)